types/wlr_screenshooter.c

```diff
@@ -144,6 +160,25 @@ static void screenshooter_bind(struct wl_client *wl_client, void *data,
 		screenshooter, NULL);
 }
 
+void wlr_screenshooter_destroy(struct wlr_screenshooter *screenshooter) {
```

Comment:
This and those like it should be exported so users can do it if they want to

Reply:
Already done

```c
void wlr_screenshooter_destroy(struct wlr_screenshooter *screenshooter);
```

Reply:
I'm pretty sure all destructors are exported btw